web/src/main/webapp/WEB-INF/pages/application_state.jsp

```diff
@@ -227,14 +224,40 @@
                         <div class="help-block with-errors"></div>
                       </div>
                   </c:when>
+
+                  <c:when test ="${data.stateName == 'profInterview'}">
+                     <div class="form-group">
+                        <label for="recommendedPositionLevelInput"  class="control-label col-sm-4"><spring:message code="statehistory.field.recommendedPositionLevel"/></label>
+                        <div class="col-sm-8">
+                          <p class="form-control-static <c:if test='${stat.first}'>stateData</c:if>" id="reviewerNameP">L${data.recommendedPositionLevel}</p>
```

Comment:
While `id` is `reviewerNameP`, the actual data displayed is `recommendedPositionLevel`.

Rename the id to reflect the data display in it.